taskfiles/k8s.yml

```diff
@@ -156,12 +156,23 @@ tasks:
         silent: true
       - 'echo "~~~ :docker: Build & Push operator container"'
       - docker buildx build
-        --platform linux/arm64,linux/amd64
+        --provenance false --sbom false --platform linux/amd64
         --file operator/Dockerfile
         --target manager
-        --tag docker.io/redpandadata/{{.OPERATOR_REPO}}:{{.TAG_NAME}}
+        --tag docker.io/redpandadata/{{.OPERATOR_REPO}}:{{.TAG_NAME}}-amd64
         --push
         ./dist
+      - docker buildx build
+        --provenance false --sbom false --platform linux/arm64
```

Comment:
What's the reasoning for disabling provenance and sbom? Not strong feelings, just curious if it was necessary for something.

Reply:
I borrowed the steps from @birdayz script. 

For nightly I personally wouldn't care that much, but for official container image registry it would be wise to generate sbom and sign that image. Yet it is not our standard software delivery practise as far as I know.